code/modules/mob/living/carbon/carbon.dm

```diff
@@ -89,6 +89,8 @@
 
 
 /mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, override = 0, tesla_shock = 0)
+	if(dna && dna.check_mutation(SMES_HUMAN))

```

Comment:
it would just change it to the initial of the new species, which technically isn't wrong @Core0verload